### PR TITLE
BG-50243-determine-fixes-for-breaking-change-in-bit-go-version-15-to-remain-backwards-compatible-master

### PR DESCRIPTION
**Pull request by mmcshinsky-bitgo**
BG-50243

### CODE REVIEW

File: modules/bitgo/src/v2/index.ts [hunk 1]
```diff
@@ -1,3 +1,17 @@
+/* eslint-disable import/no-internal-modules */
+/**
+ * @deprecated after version 14.0.0
+ * Retained for short term backwards compatibility - migrate to: @bitgo/sdk-api and @bitgo/sdk-core
+ */
+export * from '@bitgo/sdk-core/dist/src/bitgo/baseCoin/baseCoin';
```
Comment (bitgoAaron) [inline modules/bitgo/src/v2/index.ts:6]: can we just `export * from '@bitgo/sdk-core` instead of digging into dist folders?

Comment (mmcshinsky-bitgo) [reply modules/bitgo/src/v2/index.ts:6]: Yeah, it's my conflict between specific backwards compatibility or "adding more features" by exporting everything. :/

Comment (mmcshinsky-bitgo) [reply modules/bitgo/src/v2/index.ts:6]: I have some overlapping types to resolve if doing it that way :(

File: modules/bitgo/src/index.ts [hunk 2]
```diff
@@ -28,7 +29,7 @@ export { Buffer } from 'buffer';
 export { CustomSigningFunction, SignedTransaction };
```
Comment (bitgoAaron) [inline modules/bitgo/src/index.ts:29]: these & `export { tss }` below are redundant now that we're exporting * from core

File: modules/sdk-core/src/index.ts [hunk 3]
```diff
@@ -1,6 +1,8 @@
 export * from './account-lib';
+export * as accountLib from './account-lib';
 export * from './api';
 export * from './bitgo';
+export * as bitgo from './bitgo';
```
Comment (bitgoAaron) [inline modules/sdk-core/src/index.ts:5]: are these still necessary if we're `exporting * from './bitgo` ? Do we still need to namespace them?

**Review by bitgoAaron — CHANGES_REQUESTED** [review]
couple of extra exports still

File: modules/sdk-core/src/index.ts (hunk 3 above)
Comment (mmcshinsky-bitgo) [reply modules/sdk-core/src/index.ts:5]: Not anymore. I'll remove them.

**Review by bitgoAaron — APPROVED** [review]

**Pull request merged**